bin/centos71-devbuild.sh

```diff
@@ -44,9 +44,9 @@ if [ "${1}" = "--fg" ]
 then
   echo "CentOS 7.1 Dev Build kicked off, Log being saved in ${LOG_FILE} ..."
 
-  time ruby /build/scripts/vmbuild.rb $BUILD_OPTIONS 2>&1 | tee ${LOG_FILE}
+  time ruby /build/manageiq/scripts/vmbuild.rb $BUILD_OPTIONS 2>&1 | tee ${LOG_FILE}
```

Comment:
> Should this be manageiq-appliance where the scripts are moving to?

hehe, confusion already... these build scripts live in `manageiq-appliance-build` now.  

It doesn't matter what we call the directory here but it would make more sense for a new person to understand it.  git clone would do the right thing since it will clone it to /build/manageiq-appliance-build